products/cfengine.md

```diff
@@ -29,12 +37,12 @@ releases:
     latestReleaseDate: 2021-12-10
 -   releaseCycle: "3.18"
     lts: true
-    latest: "3.18.2"
+    latest: "3.18.3"
     support: 2023-06-24
-    link: https://github.com/cfengine/core/blob/3.18.2/ChangeLog
-    eol: false
+    link: https://github.com/cfengine/core/blob/3.18.3/ChangeLog
+    eol: 2023-06-24
```

Comment:
Should be 2024-06-24 according to https://cfengine.com/blog/2021/cfengine-3-18-lts-released-extensibility/.

Reply:
I will fix it.